domains/transfer/Transfer.API/API/Shared/Swagger/SwaggerDefaultValues.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Linq;
+using System.Text.Json;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace API.Shared.Swagger;
+
+public class SwaggerDefaultValues : IOperationFilter
```

Comment:
It is not obvious for me what this filter actually does. I think a comment or more saying name would help.

Reply:
It takes each generated api spec, checks if they are deprecated. If it is deprecated, it will add a description saying so. It will also add a DEFAULT_VALUE to each endpoint.

Reply:
Tried with a comment